Big Integers/Big Sorting.cpp

```diff
@@ -0,0 +1,42 @@
+/*
+Problem Statement:
+Given a list of numbers, which would even include numbers of some 20 digit numbers or so. And your task is to sort them.
+*/
+#include<iostream>
+#include<vector>
+#include<algorithm> //for sort()
+using namespace std;
+bool customSorting(string a, string b){
+    if(a.length() == b.length()) // if both the strings are of same length
+        return a < b; // check if first string is less than the second string
```

Comment:
Refactor to use [`std::string::compare`](https://www.geeksforgeeks.org/stdstringcompare-in-c/) for type safety.

Reply:
📌 Skipping this for now.